fili-wikipedia-example/src/main/java/com/yahoo/wiki/webservice/data/config/names/WikiApiDimensionConfigInfo.java

```diff
@@ -0,0 +1,65 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.wiki.webservice.data.config.names;
+
+import com.yahoo.bard.webservice.data.config.names.DimensionName;
+import com.yahoo.bard.webservice.util.EnumUtils;
+
+/**
+ * Hold all the Wikipedia API dimension names.
+ */
+public enum WikiApiDimensionConfigInfo implements DimensionName {
+    PAGE("Page is a document that is suitable for World Wide Web and web browsers", "wiki page", "General"),
+    LANGUAGE("Language used to write the wiki page", "wiki language", "General"),
+    USER("User is a person who generally use or own wiki services", "wiki user", "General"),
+    UNPATROLLED("Unpatrolled are class of pages that are not been patrolled", "wiki unpatrolled", "General"),
+    NEW_PAGE("New Page is the first page that is created in wiki ", "wiki newPage", "General"),
+    ROBOT("Robot is an tool that carries out repetitive and mundane tasks", "wiki robot", "General"),
+    ANONYMOUS("Anonymous are individual or entity whose identity is unknown", "wiki anonymous", "General"),
+    NAMESPACE("Namespace is a set of wiki pages that begins with a reserved word", "wiki namespace", "General"),
+    CONTINENT("Name of the Continent to which the wiki page belongs ", "wiki continent", "General"),
+    COUNTRY("Name of the Country to which the wiki page belongs", "wiki country", "General"),
+    REGION("Name of the Region to which the wiki page belongs", "wiki region", "General"),
+    CITY("Name of the City to which the wiki page belongs", "wiki city", "General");
+
+    private final String camelName;
+    private final String description;
+    private final String longName;
+    private final String category;
+
+    /**
+     * Constructor of Wikipedia Dimension Name.
+     *
+     * @param description  A description of the dimension and its meaning.
+     * @param longName  The Long Name is the external, end-user-facing long  name for the dimension.
+     * @param category  The Category is the external, end-user-facing category for the dimension.
+     */
+    WikiApiDimensionConfigInfo(String description, String longName, String category) {
```

Comment:
Consider get rid of the `category` property from the enum and simply return in the method using default category.

```java
    @Override
    public String getCategory() {
        return Dimension.DEFAULT_CATEGORY;
    }
```